moj-node/server/views/components/header/template.njk

```diff
@@ -6,7 +6,7 @@
         {% if params.user %}
           <div id="hub-auth-header" class="hub-header-user-container">
             <span class="govuk-!-font-weight-bold">{{ params.user.name }}</span>
-            <span>{{ params.appName }}</span>
+            <h1>{{ params.appName }}</h1>
```

Comment:
Are we sure the username / site name is a H1?

Reply:
It's just the prison name essentially. Since it's the index page there's nothing else that it can be. So it becomes:

Prison Name
- Today's Events
- News etc
...

Although I just realised we've got the `params.user` check around the whole block, when maybe it should just be around the username since the appName should always be set?

Reply:
cool